### DISCUSSION TITLE
rtorrent Stop state / move

### DISCUSSION THREAD

**Discussion by vdias** [rakshasa/rtorrent#1105] [Ideas]
Why is not possible to configure a task for stop state, just like is posible to configure for complete state...?

Why there is no time configuration for ratio?

**Comment by pyroscope** [comment]
It is. There is (though "time configuration for ratio" makes not that much sense). 

You might want define what YOU think "stop state" etc. actually means. And use more words to actually describe the wanted behavior.

**Comment by vdias** [comment]
"time configuration for ratio" makes not that much sense)."  ---   My
private tracker requests a minimum seeding time for all downloads.

The stop state is simple... I want to automate my downloads and move all
data to a folder organized by state.

1) Download data on a "incompleted" folder
2) Move data to "seeding" folder after download is finished
3) Stop, remove torrent and move data to "completed" folder after ratio is
reached

Step 1 and 2 is configured, but step 3 its just impossible... because there
is no stop state...

**Reply by pyroscope** [reply]
Ratio groups can execute any command, including another move to somewhere, and `elapsed.greater` can be used for time-based conditions. The point of rT is that you can define these things yourself to your exact liking, but that means you MUST define them yourself if nobody else did. And you need to invest the time to learn how to.

**Comment by vdias** [comment]
RTorrentRatioHandling · rakshasa/rtorrent Wiki (github.com)
<https://github.com/rakshasa/rtorrent/wiki/RTorrentRatioHandling>

No elapsed.greater here...
